instruments/thorlabs/thorlabsapt.py

```diff
@@ -1267,11 +1267,68 @@ def motor_model(self, newval):
 
         # MOTOR COMMANDS #
 
+        @property
+        def backlash_correction(self):
+            """Get the backlash correction, if stage is defined, unitful.
+
+            :return: Unitful quantity of backlash correction.
```

Comment:
Can you add what the units are (ie, `counts`)

Reply:
This now has a more useful explanation and example.